src/frontend/apps/desk/src/features/teams/components/Member/MemberGrid.tsx

```diff
@@ -0,0 +1,89 @@
+import { DataGrid, usePagination } from '@openfun/cunningham-react';
+import React, { useEffect } from 'react';
+import { useTranslation } from 'react-i18next';
+
+import IconUser from '@/assets/icons/icon-user.svg';
+import { Box, Card, TextErrors } from '@/components';
+import { useCunninghamTheme } from '@/cunningham';
+import { Team, useTeamAccesses } from '@/features/teams/api/';
+import { PAGE_SIZE } from '@/features/teams/conf';
+
+interface MemberGridProps {
+  team: Team;
+}
+
+export const MemberGrid = ({ team }: MemberGridProps) => {
+  const { t } = useTranslation();
+  const { colorsTokens } = useCunninghamTheme();
+
+  const pagination = usePagination({
+    pageSize: PAGE_SIZE,
+  });
+  const { page, pageSize, setPagesCount } = pagination;
+  const { data, isLoading, error } = useTeamAccesses({
+    teamId: team.id,
+    page,
+  });
+
+  const accesses = data?.results;
+
+  useEffect(() => {
+    setPagesCount(data?.count ? Math.ceil(data?.count / pageSize) : 0);
+  }, [data?.count, pageSize, setPagesCount]);
```

Comment:
I might be wrong, suggestion, please feel free to ignore it.
Why is `setPagesCount` a dependency ?

```suggestion
  useEffect(() => {
    if (!accesses) {
      setPagesCount(0);
    }
    setPagesCount(Math.ceil(accesses.length / pageSize));
  }, [accesses, pageSize, page]);
```

Reply:
I close it @AntoLC !